applications/leaderboard/step5/Program.cs

```diff
@@ -150,13 +150,12 @@ enum ExitCode : int
                 $"Data Source=projects/{projectId}/instances/{instanceId}"
                 + $"/databases/{databaseId}";
 
-            using (TransactionScope scope = new TransactionScope(
-               TransactionScopeAsyncFlowOption.Enabled))
+            Int64 numberOfPlayers = 0;
+            using (var connection = new SpannerConnection(connectionString))
             {
-                Int64 numberOfPlayers = 0;
-                using (var connection = new SpannerConnection(connectionString))
+                await connection.OpenAsync();
+                await connection.RunWithRetriableTransactionAsync(async (transaction) =>
                 {
-                    await connection.OpenAsync();
                     // Execute a SQL statement to get current number of records
                     // in the Players table to use as an incrementing value 
                     // for each PlayerName to be inserted.

```

Comment:
Similar comment here to get the number of players, it can all be done in one line of code.

Reply:
Done.